nautobot_golden_config/views.py

```diff
@@ -560,12 +560,7 @@ class ConfigPlanUIViewSet(views.NautobotUIViewSet):
     table_class = tables.ConfigPlanTable
     lookup_field = "pk"
     action_buttons = ("add",)
-
```

Comment:
This is the `get_form_class` from the parent class `nautobot.core.views.mixins.NautobotViewSetMixin`:

```py
    def get_form_class(self, **kwargs):
        """
        Helper function to get form_class for different views.
        """

        if self.action in ["create", "update"]:
            if getattr(self, f"{self.action}_form_class"):
                form_class = getattr(self, f"{self.action}_form_class")
            else:
                form_class = getattr(self, "form_class", None)
        elif self.action == "bulk_create":
            required_field_names = [
                field["name"]
                for field in get_csv_form_fields_from_serializer_class(self.serializer_class)
                if field["required"]
            ]

            class BulkCreateForm(BootstrapMixin, Form):
                csv_data = CSVDataField(required_field_names=required_field_names)
                csv_file = CSVFileField()

            form_class = BulkCreateForm
        else:
            form_class = getattr(self, f"{self.action}_form_class", None)

        if not form_class:
            if self.action == "bulk_destroy":
                queryset = self.get_queryset()

                class BulkDestroyForm(ConfirmationForm):
                    pk = ModelMultipleChoiceField(queryset=queryset, widget=MultipleHiddenInput)

                return BulkDestroyForm
            else:
                # Check for request first and then kwargs for form_class specified.
                form_class = self.request.data.get("form_class", None)
                if not form_class:
                    form_class = kwargs.get("form_class", None)
        return form_class
```

Reply:
yes, thanks!! This was the reason I did this to begin with, put in the fix so as you said, this no longer needed https://github.com/nautobot/nautobot/pull/4161